scm-ui/ui-extensions/src/extensionPoints.ts

```diff
@@ -509,13 +608,39 @@ type RepositoryRedirectProps = {
   };
 };
 
-export type RepositoryRedirect = ExtensionPointDefinition<"repository.redirect",
+export type RepositoryRedirect = ExtensionPointDefinition<
+  "repository.redirect",
   (props: RepositoryRedirectProps) => string,
-  RepositoryRedirectProps>;
+  RepositoryRedirectProps
+>;
+
+export type InitializationStep<
+  Step extends string | undefined = undefined
+> = SimpleRenderableDynamicExtensionPointDefinition<
+  "initialization.step.",
+  Step,
+  {
+    data: HalRepresentation;
+  }
+>;
 
-export type InitializationStep<Step extends string | undefined = undefined> =
-  SimpleRenderableDynamicExtensionPointDefinition<"initialization.step.",
-    Step,
-    {
-      data: HalRepresentation;
-    }>;
+export type ContentActionExtensionProps = {
+  repository: Repository;
+  file: File;
+  revision: string;
+  handleExtensionError: React.Dispatch<React.SetStateAction<Error | undefined>>;
+  contentType?: ContentType;
+  unmountComponent: () => void;
```

Comment:
I am unhappy with this solution but can't find a better way. Maybe discuss this with the whole team?